database/sql/src/main/scala/cromwell/database/slick/tables/MetadataEntryComponent.scala

```diff
@@ -62,10 +62,10 @@ trait MetadataEntryComponent {
   val metadataEntryIdsAutoInc = metadataEntries returning metadataEntries.map(_.metadataEntryId)
 
   val metadataEntriesForWorkflowExecutionUuid = Compiled(
-    (workflowExecutionUuid: Rep[String]) => for {
+    (workflowExecutionUuid: Rep[String]) => (for {
       metadataEntry <- metadataEntries
       if metadataEntry.workflowExecutionUuid === workflowExecutionUuid
-    } yield metadataEntry
+    } yield metadataEntry).sortBy(_.metadataTimestamp)
```

Comment:
One plus side is that it simplifies the metadata building code which doesn't have to carry along the timestamp in order to be able to compare with another value when / if needed.